[2023-06-03T17:24:46.124+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: most_popular_movies.scrape_most_popular_titles manual__2023-06-03T17:24:41.517975+00:00 [queued]>
[2023-06-03T17:24:46.130+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: most_popular_movies.scrape_most_popular_titles manual__2023-06-03T17:24:41.517975+00:00 [queued]>
[2023-06-03T17:24:46.130+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-06-03T17:24:46.141+0000] {taskinstance.py:1327} INFO - Executing <Task(_PythonDecoratedOperator): scrape_most_popular_titles> on 2023-06-03 17:24:41.517975+00:00
[2023-06-03T17:24:46.146+0000] {standard_task_runner.py:57} INFO - Started process 1592 to run task
[2023-06-03T17:24:46.150+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'most_popular_movies', 'scrape_most_popular_titles', 'manual__2023-06-03T17:24:41.517975+00:00', '--job-id', '25', '--raw', '--subdir', 'DAGS_FOLDER/most_popular_movies.py', '--cfg-path', '/tmp/tmpqn32_i9n']
[2023-06-03T17:24:46.150+0000] {standard_task_runner.py:85} INFO - Job 25: Subtask scrape_most_popular_titles
[2023-06-03T17:24:46.181+0000] {task_command.py:410} INFO - Running <TaskInstance: most_popular_movies.scrape_most_popular_titles manual__2023-06-03T17:24:41.517975+00:00 [running]> on host 8406b7b18d2b
[2023-06-03T17:24:46.246+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='most_popular_movies' AIRFLOW_CTX_TASK_ID='scrape_most_popular_titles' AIRFLOW_CTX_EXECUTION_DATE='2023-06-03T17:24:41.517975+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-06-03T17:24:41.517975+00:00'
[2023-06-03T17:24:46.248+0000] {taskinstance.py:1824} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/decorators/base.py", line 220, in execute
    return_value = super().execute(context)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 181, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 198, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/most_popular_movies.py", line 28, in scrape_most_popular_titles
    movie_names = _scrape_most_popular_titles(soup)
  File "/opt/airflow/dags/helper/web_scrape.py", line 45, in _scrape_most_popular_titles
    titlesRefs = soup.find_all('td', {'class':'titleColumn'})
AttributeError: 'NoneType' object has no attribute 'find_all'
[2023-06-03T17:24:46.253+0000] {taskinstance.py:1350} INFO - Marking task as FAILED. dag_id=most_popular_movies, task_id=scrape_most_popular_titles, execution_date=20230603T172441, start_date=20230603T172446, end_date=20230603T172446
[2023-06-03T17:24:46.261+0000] {standard_task_runner.py:109} ERROR - Failed to execute job 25 for task scrape_most_popular_titles ('NoneType' object has no attribute 'find_all'; 1592)
[2023-06-03T17:24:46.280+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 1
[2023-06-03T17:24:46.293+0000] {taskinstance.py:2651} INFO - 0 downstream tasks scheduled from follow-on schedule check
